src/view.c

```diff
@@ -1869,6 +1884,9 @@ view_destroy(struct view *view)
 		if (rc.adaptive_sync == LAB_ADAPTIVE_SYNC_FULLSCREEN) {
 			wlr_output_enable_adaptive_sync(view->output->wlr_output, false);
 		}
+		if (rc.allow_tearing == LAB_TEARING_FULLSCREEN) {
+			view->output->tearing = false;
+		}
```

Comment:
I'm not sure this is required if you use `set_tearing` to force the right value every time (*i.e.*, after every window activation or full-screen state switch). I'm also not sure this properly handles state transitions in some corner cases where, e.g., you have other full-screen views open that should still force tearing but may not be active.

Reply:
Well, i wouldn't think anyone would care as tearing should be reactivated when the view becomes active.

Setting it that way sounds like good idea, at least in terms of keeping code simple.